### PR TITLE
RSVP: Remove dotted border when the icon wrapper takes the whole width

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/ 118952

### CODE REVIEW

File: src/styles/rsvp/frontend.pcss [hunk 1]
```diff
@@ -36,6 +36,10 @@
 	svg {
 		width: 60px;
 	}
+
+	@media only screen and (max-width: 575px) {
```
Comment (paulmskim) [inline src/styles/rsvp/frontend.pcss]: better if you do mobile first for css, using `min-width` rather than the `max-width` media query.

Comment (juanfra) [reply src/styles/rsvp/frontend.pcss]: Ok, made the changes.

Comment (paulmskim) [reply src/styles/rsvp/frontend.pcss]: Awesome. Just one more thing to be aware of: when using `min-width`, in this case at `575px`, the styles in the media query take effect at and greater than `575px` screen width (so it includes `575px`. However, with how you had it at `max-width`, the styles take effect at and lesser than `575px` screen width. So in one case, the smaller viewport includes `575px`, the other case the larger viewport includes `575px`. I'd check to make sure that at `575px` screen width, the style is what you expect it to be.

**Comment by juanfra** [comment]
Thanks, missed that.

**Pull request merged**